tests/functional/modules/test_zos_archive_func.py

```diff
@@ -84,11 +85,13 @@ def create_multiple_data_sets(ansible_zos_module, base_name, n, type, ):
 def create_multiple_members(ansible_zos_module, pds_name, member_base_name, n):
     test_members = []
     for i in range(n):
-        curr_ds = dict(name="{0}({1})".format(pds_name, member_base_name+str(i)),
-                       type="member",
-                       state="present",
-                       replace=True,
-                       force=True)
+        curr_ds = {
+            "name":F"{pds_name}({member_base_name+str(i)})",
```

Comment:
I know is possible to python to recognize f-strings with capital F but seems weird. 
```suggestion
            "name":f"{pds_name}({member_base_name+str(i)})",
```

Reply:
Typo I suppose